src/main/scala/firrtl/annotations/transforms/EliminateTargetPaths.scala

```diff
@@ -67,7 +67,7 @@ object EliminateTargetPaths {
     }
     def onStmt(s: Statement): Statement = s map onStmt match {
       case w@WDefInstance(info, name, module, tpe) if toRename.contains(module) => w.copy(module = toRename(module))
-      case w@DefInstance(info, name, module) if toRename.contains(module) => w.copy(module = toRename(module))
+      case w@DefInstance(info, name, module, _) if toRename.contains(module) => w.copy(module = toRename(module))
```

Comment:
You can actually drop this line entirely (or the one above it) for the `1.4.0` version. The WDefInstance unapply will match the same stuff.

Reply:
I don't know if you're going to backport this PR (looks like it's marked 1.4.0), but make sure to put it back in if you do :)

Reply:
cool, glad that stuff is cleaned up now!